<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="statusTip">
   <string extracomment="Rysuje sygnał dla obecnego szeregu fouriera"/>
  </property>
  <property name="whatsThis">
   <string extracomment="Rysuje sygnał dla obecnego szeregu fouriera"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_generate">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>440</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Generuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyczyść</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>320</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ilość harmonicznyc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>350</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>częstotliwość</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>901</width>
      <height>291</height>
     </rect>
    </property>
    <property name="statusTip">
     <string comment="Rysuje sygnał dla obecnego szeregu fouriera"/>
    </property>
    <property name="accessibleName">
     <string>Rysuje sygnał dla obecnego szeregu fouriera</string>
    </property>
    <property name="accessibleDescription">
     <string>Rysuje sygnał dla obecnego szeregu fouriera</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="t_0">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>30</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="points">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="delta_t">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>60</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>czas początkowy</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>przedział czasu</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>90</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ilość punktów</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>901</width>
      <height>211</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>harmonic number</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>sine_coeff</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>cosine_coeff</string>
     </property>
    </row>
   </widget>
   <widget class="QPushButton" name="btn_plot">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>120</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string comment="Rysuje sygnał dla obecnego szeregu fouriera" extracomment="Rysuje sygnał dla obecnego szeregu fouriera"/>
    </property>
    <property name="statusTip">
     <string comment="Rysuje sygnał dla obecnego szeregu fouriera"/>
    </property>
    <property name="whatsThis">
     <string comment="Rysuje sygnał dla obecnego szeregu fouriera" extracomment="Rysuje sygnał dla obecnego szeregu fouriera">Rysuje wykres przebiegu dla obecnego szeregu fouriera</string>
    </property>
    <property name="accessibleDescription">
     <string comment="Rysuje sygnał dla obecnego szeregu fouriera" extracomment="Rysuje sygnał dla obecnego szeregu fouriera"/>
    </property>
    <property name="text">
     <string>Rysuj</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_maxHarmonic">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>320</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_freq">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>350</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_waveType">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>410</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Sine</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pulse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Triangle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sawtooth</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>410</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>rodzaj sygnału</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_amp">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>380</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>380</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>amplituda</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>500</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Wczytaj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_addHarmonic">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>470</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="statusTip">
     <string comment="Rysuje sygnał dla obecnego szeregu fouriera" extracomment="Rysuje sygnał dla obecnego szeregu fouriera"/>
    </property>
    <property name="whatsThis">
     <string comment="Rysuje sygnał dla obecnego szeregu fouriera"/>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionget_info">
   <property name="text">
    <string>get info</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
